samples/batch.js

```diff
@@ -0,0 +1,246 @@
+/**
+ * Copyright 2018, Google, Inc.
```

Comment:
I think it should be Google LLC here and everywhere.

Reply:
Is this something we need to change across all files? If so, does something of that scale belong in a PR like this?

Reply:
I meant only new `*.js` files introduced within this PR, of course. But probably it's OK to merge as is.